bundle/manifests/update-service-operator.clusterserviceversion.yaml

```diff
@@ -40,9 +40,31 @@ spec:
         name: ""
         version: ""
       specDescriptors:
-      - path: replicas
+      - description: graphDataImage is a container image that contains the UpdateService
+          graph data.
+        displayName: Graph Data Image
+        path: graphDataImage
+      - description: releases is the repository in which release images are tagged,
+          such as quay.io/openshift-release-dev/ocp-release.
+        displayName: Releases
+        path: releases
+      - description: replicas is the number of pods to run. When >=2, a PodDisruptionBudget
+          will ensure that voluntary disruption leaves at least one Pod running at
+          all times.
+        displayName: Replicas
+        path: replicas
       statusDescriptors:
-      - path: replicas
```

Comment:
I believe this was a mistake from https://github.com/openshift/cincinnati-operator/pull/131/files#diff-ef93a20b4f66d5fb3e6d84caf1169c43d4f750684fabd1a7c27065312dbb7fcdR26-R27